test/OWNERS: add pohly as approver

pohly opened the pull request:
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

As technical lead in SIG Testing it would be useful to have the right to approve test changes...

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

/assign @aojea

pohly committed:
test/OWNERS: add pohly as approver

As technical lead in SIG Testing it would be useful to have the right to
approve test changes...

aojea commented:
I think we should just add ourselves to the corresponding aliases https://github.com/kubernetes/kubernetes/blob/1edbb8cf1a864c2dd71a1e5b7282de4bc3d8f5fa/OWNERS_ALIASES#L214-L222 

sig-testing-reviewers and sig-testing-approvers will give you the same rights (well earned)

pohly commented:
`sig-testing-reviewers` isn't used for `test`, so I added myself there as before. For approval rights I modified the alias and also cleaned up a bit while at it.

aojea commented:
/lgtm
/assign @BenTheElder @dims @spiffxp

dims commented on test/OWNERS:
```diff
@@ -10,20 +10,17 @@ reviewers:
-  - cblecker
   - dchen1107
   - deads2k
-  - dims
```
Am i losing my privileges @pohly ?

pohly replied:
Oops, yes, you are.

I thought you were in sig-testing-approvers, but you aren't. That was only sig-testing-reviewers.

Fixed.

dims replied:
> Oops, yes, you are.

thanks @pohly !

dims commented:
/approve
/lgtm
/hold for additional folks to chime in

BenTheElder commented on OWNERS_ALIASES:
```diff
@@ -212,13 +212,17 @@ aliases:
     - dims
+    - pohly
+    - spiffxp
   sig-testing-approvers:
```
I think we should deprecate this alias instead, it only has a few people and we don't need to apply it to many directories.

BenTheElder replied:
(And aliases obscure ownership)

pohly replied:
I agree and removed it in the first commit of this PR, then added me in the second.

pohly replied:
@dims: do you agree? Can you re-add the LGTM and perhaps approval?

pohly committed:
OWNERS: remove sig-testing-approvers

It's not used enough to add any real value. Instead, it obscures the real
ownership when someone looks at an OWNERS file where it is used.

dims commented:
/approve
/lgtm

BenTheElder reviewed (commented):
/lgtm
/approve
/priority important-longterm
/triage accepted

BenTheElder commented:
/hold cancel

Thanks for all your work so far and thanks for cleaning that alias up!

aojea commented:
> /lgtm

👏
